.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.tabSize": 2
```

Comment:
Do you want this to be part of this CL?

Reply:
My bad, this was accidentally added as one of the commit. This should be an outdated file now.